R/expect-condition.R

```diff
@@ -301,6 +297,13 @@ cnd_matcher <- function(base_class,
   check_string(class, allow_null = TRUE, call = error_call)
   check_string(pattern, allow_null = TRUE, allow_na = TRUE, call = error_call)
 
+  if (is.null(pattern) && dots_n(...) > 0) {
+    cli::cli_abort(
+      "{.arg ...} ignored when {.arg pattern} is not set.",
+      call = error_call
```

Comment:
Yea I see `error_call = trace_env` in the caller of `cnd_matcher()` above, but I see that https://github.com/r-lib/testthat/pull/2012/ fixes that, so 👍

Reply:
And it worked, yay!